test/server/lib/GranularAccess.ts

```diff
@@ -457,6 +457,52 @@ describe('GranularAccess', function() {
     ]);
   });
 
+  it('respects SCHEMA_EDIT when converting a column', async () => {
+    // Initially, schema flag defaults to ON for editor.
+    await freshDoc();
+    await owner.applyUserActions(docId, [
+      ['AddTable', 'Table1', [{id: 'A', type: 'Int'},
+                              {id: 'B', type: 'Int'},
+                              {id: 'C', type: 'Int'}]],
+      ['AddRecord', '_grist_ACLResources', -1, {tableId: 'Table1', colIds: 'C'}],
+      ['AddRecord', '_grist_ACLRules', null, {
+        resource: -1, aclFormula: 'user.Access == OWNER', permissionsText: '-R',
```

Comment:
Why do you add this rule, I don't think  it is exersiced anywhere below?

Reply:
We need at least one rule. In the absence of any rules, `SPECIAL_ACTIONS` are already allowed for editors, so the test would succeed trivially.

Added a comment.